package-lock.json

```diff
@@ -47,6 +47,61 @@
                 "webpack-merge": "^5.10.0"
             }
         },
+        "../mynah-ui": {
```

Comment:
This looks like it shouldn't be here. Might be causing the build to fail?

Reply:
100% right, thanks